e2etests/README_cucumber.md

```diff
@@ -1,6 +1,13 @@
-# Department for Education
+# Test Automation (Functional)
```

Comment:
DfE is large I still think this should have a reference to the project name.

Reply:
No it should not

By that logic, every single file would have DfE in it

Redundant